themes/bootstrap3/templates/holds/list.phtml

```diff
@@ -28,7 +28,7 @@
           <div class="checkbox">
             <label>
               <input type="checkbox" name="selectAll" class="checkbox-select-all">
-              <?=$this->transEsc('select_page')?>
+              <?=$this->transEsc('select_all_on_page')?>
```

Comment:
There is no select_all_on_page key in the language file. Should this remain as select_page, or should select_page be copied or renamed in the language file?